### PR TITLE
Test against Plone 5.1.x

### PR DESCRIPTION
**Pull request by mbaechtold**

### CODE REVIEW

File: test-plone-5.1.x.cfg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[buildout]
+extends =
+    https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
+    sources.cfg
+
+package-name = ftw.publisher.receiver
+
+
+[branches]
+ftw.publisher.core = ne/plone-5.1
```
Comment (mbaechtold) [inline test-plone-5.1.x.cfg]: 🚧

**Comment by mbaechtold** [comment]
I have added a changelog entry.

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ Changelog
 2.2.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Test against Plone 5.1. [mbaechtold]
```
Comment (jone) [inline docs/HISTORY.txt]: I'd say the package supports Plone 5 now, not only its tests, does it? 😉

**Comment by mbaechtold** [comment]
@jone, are we done here?

File: development.cfg [hunk 3]
```diff
@@ -1,4 +1,4 @@
 [buildout]
 extends =
-    test-plone-4.3.x.cfg
+    test-plone-5.1.x.cfg
```
Comment (jone) [inline development.cfg]: Should we already switch development to 5.1?
I'd probably rather keep it at 4.3 for now..

Comment (mbaechtold) [reply development.cfg]: Good idea. I have changed that back to 4.3.x.

**Pull request merged**